lib/rules/spaced-comment.js

```diff
@@ -265,11 +295,37 @@ module.exports = {
         }
 
         /**
+         * Reports an ending spacing error with an appropriate message.
+         * @param {ASTNode} node - A comment node to check.
+         * @param {string} message - An error message to report.
+         * @param {string} match - An array of the matched whitespace characters.
+         * @returns {void}
+         */
+        function reportEnd(node, message, match) {
+
+            context.report({
+                node: node,
+                fix: function(fixer) {
+                    if (requireSpace) {
+                        return fixer.insertTextAfterRange([node.start, node.end - 2], " ");
+                    } else {
+                        var end = node.end - 2,
+                            start = end - match[0].length;
+
+                        return fixer.replaceTextRange([start, end], "");
+                    }
+                },
+                message: message
+            });
+        }
+
+        /**
          * Reports a given comment if it's invalid.
          * @param {ASTNode} node - a comment node to check.
          * @returns {void}
          */
         function checkCommentForSpace(node) {
+
```

Comment:
@mikesherov you can feel free to propose it in an issue. Let's not couple with this discussion.